frontend/app/views/spree/checkout/_confirm.html.erb

```diff
@@ -6,6 +6,13 @@
 
 <br />
 
+<% if Spree::Config[:require_terms_and_conditions] %>
+  <p class="field" style='clear: both'>
+    <%= form.checkbox :terms_and_conditions %>
+    <%= form.label :terms_and_conditions, Spree.t(:accept_terms_and_conditions) %>
```

Comment:
That's not true. Rails adds a `for="id-of-checkbox"` to the label. So this works fine ;)